src/DynamoCore/UpdateManager/UpdateManager.cs

```diff
@@ -654,20 +654,34 @@ public void QuitAndInstallUpdate()
 
         public void HostApplicationBeginQuit()
         {
-            if (!string.IsNullOrEmpty(UpdateFileLocation))
+            if (string.IsNullOrEmpty(UpdateFileLocation)) return;
+
+            var currDir = Path.GetDirectoryName(Assembly.GetExecutingAssembly().Location);
+            var updater = Path.Combine(currDir, "InstallUpdate.exe");
```

Comment:
Consider placing "InstallUpdate.exe" in a `private const` field for increased visibility.